node/pkg/driver/node_utils.go

```diff
@@ -52,6 +54,7 @@ const (
 	resizeFsTimeoutMilliseconds = 30 * 1000
 	TimeOutMultipathdCmd        = 10 * 1000
 	multipathdCmd               = "multipathd"
+	MIN_FILES_IN_NON_EMPTY_DIR  = 1
```

Comment:
I now see the const case convention is different, sorry for [misleading](https://github.com/IBM/ibm-block-csi-driver/pull/343#discussion_r659886887).
please align this new const according to existing convention